BensonWollmanWeselyZombieProject/src/main/java/com/mycompany/bensonwollmanweselyzombieproject/Main.java

```diff
@@ -24,5 +26,76 @@ public static void main() {
         //Start Fight
         human1.attack_enemy(zombie1);
         zombie1.attack_enemy(human1);
+         */
+
+
+        /**
+         * Generate number of each unit and display to user
+         */
+        int numberOfHumans = (int)((Math.random() * 10) + 1);

```

Comment:
Can we change this to numberOfSurvivors so we stay consistent?

Reply:
This was setup to be random to align with the original assignment from the first week: 

"...The program randomly generates an array of survivors and zombies..."

If we do change it away from random, I think we should check with the prof.

Reply:
All I'm suggesting is changed the name of the variable to include survivors instead of humans

Reply:
Sorry, misunderstood.  Agree - that helps keep terminology consistent.